packages/flutter_tools/lib/src/web/compile.dart

```diff
@@ -126,10 +126,17 @@ class WebBuilder {
     } finally {
       status.stop();
     }
+    BuildEvent(
+      'web-compile',
+      type: 'web',
+      settings: 'wasm-compile: ${compilerConfig.isWasm}',
+      flutterUsage: _flutterUsage,
+    ).send();
+
     _flutterUsage.sendTiming(
       'build',
-      compilerConfig.isWasm ? 'dart2wasm' : 'dart2js',
-      Duration(milliseconds: sw.elapsedMilliseconds),
+      'compile',
```

Comment:
nit: This is a bit too generic. Maybe go back to `dart2wasm` or `dart2js` ?

Reply:
we don't KNOW what compiler was used. I could use `web-compile`?

Reply:
Er, I guess we do. Hrm. I guess I could revert my revert.